docs/src/pages/AndroidTextFieldPage/AndroidTextFieldPage.js

```diff
@@ -55,6 +55,7 @@ const AndroidTextFieldPage = () => (
     screenshots={components}
     readme={readme}
     documentationId="net.skyscanner.backpack.text/-bpk-text-field"
+    githubPath="text"
```

Comment:
This seems wrong?

Reply:
It's not — text field is inside the `text` folder